doc/man3/TS_VERIFY_CTX_set_certs.pod

```diff
@@ -0,0 +1,62 @@
+=pod
+
+=head1 NAME
+
+TS_VERIFY_CTX_set_certs, TS_VERIFY_CTS_set_certs
+- set certificates for TS response verification
+
+=head1 SYNOPSIS
+
+ #include <openssl/ts.h>
+
+ STACK_OF(X509) *TS_VERIFY_CTX_set_certs(TS_VERIFY_CTX *ctx,
+                                         STACK_OF(X509) *certs);
+ STACK_OF(X509) *TS_VERIFY_CTS_set_certs(TS_VERIFY_CTX *ctx,
+                                         STACK_OF(X509) *certs);
+
+=head1 DESCRIPTION
+
+The Time-Stamp Protocol (TSP) is defined by RFC 3161. TSP is a protocol used to
+provide long term proof of the existence of a certain datum before a particular
+time. TSP defines a Time Stamping Authority (TSA) and an entity who shall make
+requests to the TSA. Usually the TSA is denoted as the server side and the
+requesting entity is denoted as the client.
+
+In TSP, when a server is sending a response to a client, the server normally
+needs to sign the response data - the TimeStampToken (TST) - with its private
+key. Then the client shall verify the received TST by the server's certificate
+chain.
+
+TS_VERIFY_CTX_set_certs() is used to set the server's certificate chain when
+verifying a TST. B<ctx> is the verification context created in advance and
+B<certs> is a stack of B<X509> certificates.
+
+TS_VERIFY_CTS_set_certs() is a misspelled version of TS_VERIFY_CTX_set_certs()
+which takes the same parameters and returns the same result.
+
+=head1 NOTES
+
+The spelling of TS_VERIFY_CTX_set_certs() was corrected in OpenSSL 3.0.0.
+The misspelled version TS_VERIFY_CTS_set_certs() has been retained for
+compatibility reasons, but it is deprecated in OpenSSL 3.0.0.
```

Comment:
Sorry for bothering you one more time: It occurred to me only now that you mention the deprecation in both the NOTES and HISTORY section. I think we normally put it into the HISTORY section, see `git grep`:


```make

~/openssl/openssl$ git --no-pager grep -n -B 3 'deprecated in' -- doc/man3 | grep head1
doc/man3/BIO_connect.pod-102-=head1 HISTORY
doc/man3/CONF_modules_free.pod-44-=head1 HISTORY
doc/man3/DH_generate_parameters.pod-145-=head1 HISTORY
doc/man3/DSA_generate_parameters.pod-112-=head1 HISTORY
doc/man3/ENGINE_add.pod-652-=head1 HISTORY
doc/man3/ERR_error_string.pod-68-=head1 HISTORY
doc/man3/ERR_load_crypto_strings.pod-41-=head1 HISTORY
doc/man3/ERR_remove_state.pod-36-=head1 HISTORY
doc/man3/OBJ_nid2obj.pod-177-=head1 HISTORY
doc/man3/OPENSSL_config.pod-73-=head1 HISTORY
doc/man3/RAND_add.pod-96-=head1 HISTORY
doc/man3/RAND_cleanup.pod-32-=head1 HISTORY
doc/man3/RSA_generate_key.pod-96-=head1 HISTORY
doc/man3/SSL_COMP_add_compression_method.pod-92-=head1 HISTORY
doc/man3/SSL_library_init.pod-43-=head1 HISTORY
doc/man3/TS_VERIFY_CTX_set_certs.pod-48-=head1 HISTORY
doc/man3/X509_VERIFY_PARAM_set_flags.pod-369-=head1 HISTORY

...

```


So my suggestion would be: move lines 39-42 down to the HISTORY section (replacing lines 50-51) and delete the NOTES section.